app/components/archive-bucket-dialog/archive-bucket-dialog.html

```diff
@@ -22,7 +22,7 @@
       </p>
 
       <div class="archive-bucket-dialog__final-caution">
-        Caution: This cannot be undone
+        Info: This bucket can be unarchived later
       </div>
```

Comment:
i think if we change the not fully funded buckets status to draft, then i think this problem will be solved